mono/metadata/handle.h

```diff
@@ -432,7 +432,7 @@ typedef struct _MonoTypeofCastHelper *MonoTypeofCastHelper; // a pointer type un
 #ifndef MONO_HANDLE_TRACK_OWNER
 
 #define MONO_HANDLE_NEW(type, object) \
```

Comment:
There are missing parens elsewhere i.e. non-typesafe handles, but seemingly we prefer it that way???